spec/controllers/users_controller_spec.rb

```diff
@@ -1,5 +1,270 @@
 require 'rails_helper'
 
 RSpec.describe UsersController, type: :controller do
+  describe "GET 'show'" do
+    it "renders the show view" do
+      get :show
+      expect(subject).to render_template :show
+    end
 
+    let!(:user) { User.find_or_create_from_omniauth(OmniAuth.config.mock_auth[:twitter]) }
+    let(:subscription) {Subscription.create(
+            username: "Schwarzenegger", uid: "12044602" , provider: "twitter", avatar_url: "https://pbs.twimg.com/profile_images/665340796510466048/-nsoU1Q5.jpg"
+            )}
+    let(:subscription_2) {Subscription.create(
+            username: "Poodle", uid: "4943031" , provider: "vimeo", avatar_url: "https://pbs.twimg.com/profile_images/665340796510466048/-nsoU1Q5.jpg"
+            )}
+
+    let(:subscription_3) {Subscription.create(
+            username: "schwarzenegger", uid: "198945880" , provider: "instagram", avatar_url: "https://scontent-sea1-1.cdninstagram.com/hphotos-xta1/t51.2885-19/11373921_1614186788830308_1200274240_a.jpg"
+            )}
+    let(:story) {Story.create(uid: 1, text: "blah", subscription_id: 1, post_time: Time.now)}
+    let(:story) {Story.create(uid: 2, text: "blah", subscription_id: 2, post_time: Time.now)}
+    let(:story) {Story.create(uid: 3, text: "blah", subscription_id: 3, post_time: Time.now)}
+
```

Comment:
Have you all learned FactoryGirl yet? It makes creating instances for testing much easier. I'd recommend looking into it.